package.json

```diff
@@ -217,6 +217,7 @@
     "benchmark": "^2.1.4",
     "chai": "^4.1.2",
     "check-export-map": "^1.3.1",
+    "core-js": "^3.38.1",
```

Comment:
We rely on `core-js` in our polyfill tests:

https://github.com/preactjs/preact/blob/c9846f13393baef888629c219367ba23c5f2e739/test/polyfills.js#L2

Undeclared dep, hence the additon here.